src/ziggurat/messaging/producer.clj

```diff
@@ -214,17 +218,17 @@
         dead-letter-exchange-name (prefixed-queue-name topic-entity dead-letter-exchange)]
     (create-and-bind-queue queue-name exchange-name dead-letter-exchange-name)))
 
-(defn- make-delay-queue-with-retry-count [topic-entity retry-count exponential-backoff-count]
+(defn- make-delay-queue-with-retry-count [topic-entity retry-count]
   (let [{:keys [queue-name exchange-name dead-letter-exchange]} (:delay (rabbitmq-config))
         queue-name                (delay-queue-name topic-entity queue-name)
         exchange-name             (prefixed-queue-name topic-entity exchange-name)
         dead-letter-exchange-name (prefixed-queue-name topic-entity dead-letter-exchange)
-        sequence                  (if (<= retry-count exponential-backoff-count) (inc retry-count) (inc exponential-backoff-count))]
```

Comment:
exponential-backoff-count also used to control if we want to have exponential retry initially but after some retry it will be linear retry. how we can achieve it now?

Reply:
we're not supporting that anymore, I cannot think of a usecase where we'll need both linear and exponential retry in the same flow.

Reply:
IMO, it unnecessarily complicates things.